capstone/capdb/models.py

```diff
@@ -747,32 +743,31 @@ def create_or_update_metadata(self, update_existing=True, save_self=True):
                     court.jurisdiction_id = case_metadata.jurisdiction_id
                     court.save()
 
+        case_metadata.save()
+
         ### Handle citations
 
-        citations = list()
+        #  delete existing citations
+        #  create new citations
+        Citation.objects.filter(case_id=case_metadata.id).delete()
```

Comment:
I have a suggestion for how to handle this bit to avoid recreating cites that already exist. I'll send a PR to your PR ...